examples/blog/theme.config.jsx

```diff
@@ -14,6 +17,9 @@ export default {
       </h1>
     )
   },
+  cusdis: {
+    appId: 'a2d11511-7012-4254-9483-cb49c8f4dfe8'
```

Comment:
Is this `appId` for testing only?

Reply:
Yeah, I found in Google just for rendering comments in examples